src/lib/resources/app/flags.tsx

```diff
@@ -322,7 +338,11 @@ export async function autofillFlags(
 
   // Shop Language Code
   if (necessaryFlags.includes("shop-lang") && !flags["shop-lang"]) {
-    flags["shop-lang"] = "de-DE";
+    if (appName.toLowerCase().includes("magento")) {
```

Comment:
This seems like a code smell; adding exceptions for special cases in a generic module such as this has the potential to become an unmaintainable mess in the future. Maybe we can add the option to specify default/fallback values in the `AppInstaller` instance, without having to hardcode them in the `buildFlagsWithDescription` function (maybe `buildFlagsWithDescription` could accept the default values as a parameter, which gets passed in from the `AppInstaller`?)